main.go

```diff
@@ -59,35 +111,42 @@ func main() {
 		failf("Failed to fetch variants, error: %s", err)
 	}
 
-	filteredVariants := variants.Filter(config.Variant)
```

Comment:
I think gradleProject is too complicated we use GetModule to create a new moduleModel, which only purpose is to get the Task. The task's only responsibility is to get its variant.
Why not gradleProject.GetTasks() map[string][]string, GetTasks() would return the task list mapped by module names.